playground/templates/worksheet.html

```diff
@@ -339,6 +345,29 @@
 };
 
 //
+// Define what it means to show "successors" to a node.
+//
+function successors(id) {
+  // Display the node's details in the inspector "Details" panel.
+  $.getJSON(`successors/${id}`, function(result) {
+    for (let n of result.nodes) {
+      add_node(n, worksheet, null);
+    }
+
+    let elements = worksheet.elements();
+    for (let e of result.edges) {
+      add_edge(e, worksheet);
+    }
+
+    layout(worksheet, 'dagre');
```

Comment:
I think that we don't want to automatically re-layout the graph, instead leaving that as an explicit step: the analyst might possibly be using a different layout algorithm or just not want to be surprised by their view changing out from under them?